### PR TITLE
QPPCT-613: wrapper metadata exclusion test

### PR DESCRIPTION
**Pull request by clydet**
### Information
- QPPCT-613

### Changes proposed in this PR:
- added means for encodeLabel collision resolution for unnamed metadata holders

### Checklist 
- [x] All JUnit tests pass (`mvn clean verify`).
- [x] New unit tests written to cover new functionality.
- [x] Added and updated JavaDocs for non-test classes and methods.
- [x] No local design debt. Do you feel that something is "ugly" after your changes?
- [ ] Updated documentation (`README.md`, etc.) depending if the changes require it.

### CODE REVIEW

File: converter/src/main/java/gov/cms/qpp/conversion/encode/AciSectionEncoder.java [hunk 1]
```diff
@@ -119,7 +119,7 @@ private void encodeReportingParameter(JsonWrapper wrapper, Node node) {
 			DEV_LOG.error("Missing Reporting Parameters from ACI Section");
 			return;
 		}
-		reportingParamEncoder.encode(wrapper, reportingChild);
+		reportingParamEncoder.encode(wrapper, reportingChild, false);
```
Comment (halprin) [inline converter/src/main/java/gov/cms/qpp/conversion/encode/AciSectionEncoder.java]: Do you think we run into the same problem for the IA and Quality sections too?

Comment (clydet) [reply converter/src/main/java/gov/cms/qpp/conversion/encode/AciSectionEncoder.java]: Yes, those sections would also be vulnerable

Comment (halprin) [reply converter/src/main/java/gov/cms/qpp/conversion/encode/AciSectionEncoder.java]: Should we address them in this PR or a different PR?  I think these changes would be easy to extend to the other sections.

Comment (clydet) [reply converter/src/main/java/gov/cms/qpp/conversion/encode/AciSectionEncoder.java]: They both extend from AciSectionEncoder

Comment (halprin) [reply converter/src/main/java/gov/cms/qpp/conversion/encode/AciSectionEncoder.java]: Oh......

Not very bright

**Pull request merged**